tests/fix_db.py

```diff
@@ -211,8 +211,8 @@ def conn_cls(session_dsn):
     return cls
 
 
-@pytest.fixture(scope="session")
-def aconn_cls(session_dsn):
+@pytest.fixture
+def aconn_cls(dsn, anyio_backend):
```

Comment:
It seems that this fixture can be defined `scope=function` (the default) but `conn_cls` cannot, it must be `session`. I'd prefer to keep the symmetry: conceptually these object don't change across tests in the same session. Can you define `anyio_backend` as session-scoped or there are problems doing so?

Reply:
I'll give it a try; thanks for the review!

Reply:
Seems to work with session-scope.